tasks/section5.yml

```diff
@@ -131,15 +151,32 @@
       - rule_5.1.7
 
 - name: "SCORED | 5.1.7 | PATCH | Ensure permissions on /etc/cron.d are configured"
-  command: /bin/true
+  file:
+      dest=/etc/cron.d
+      state=directory
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md